.devops/rulesets.yml

```diff
@@ -0,0 +1,106 @@
+
+schedules:
+  - cron: "30 23 * * 6"
+    displayName: Build
+    always: true
+    branches:
+      include:
+        - dev
+
+variables:
+  - name: version
+    value: "3.8.8"
```

Comment:
Why not 3.9.9?  Or was there an issue with later versions? I think @kthatipally told me something once...

Extra side note, we should just add mvnw to the Windup build and not have to worry about this.  I can pair with someone to do that quickly if that's desirable (it would also change need to use Maven@4 task).

Reply:
This is WIP. All suggestions are welcome 😄 

I need to check this YAML into the repo since I cannot run ADO pipelines from a fork. I could point this to a different branch, but I do not have those permissions.

Reply:
Also, I do see a mvnw checked into the repo already. Could we use that? Assumption is yes.

I will also update the Maven task.

Reply:
I was provided with access to the repo recently. Once I verify that I can create a branch to develop on I will close this PR.

Apologies for the noise.